src/cli.yaml

```diff
@@ -0,0 +1,395 @@
+name: git-dit
+version: 0.1.0
+authors:
+    - Julian Ganz <neither@nut.email>
+    - Matthias Beyer <mail@beyermatthias>
+about: Distributed issue tracker for git
+args:
+    - verbose:
+        short: v
+        long: verbose
+        multiple: false
+        help: Print what you are doing
+    - debug:
+        short: d
+        long: debug
+        multiple: false
+        help: Print debug output
+    - trace:
+        long: trace
+        multiple: false
+        help: Print trace output (very verbose debug logging)
+subcommands:
+    - check-message:
+        about: This command checks the message in file for validity
+        version: 0.1.0
+        authors:
+            - Matthias Beyer <mail@beyermatthias.de>
+            - Julian Ganz <neither@nut.email>
+        args:
+            - filename:
+                help: Path to file to check
+                index: 1
+                takes_value: true
+                multiple: false
+
+    - create-message:
+        about: >
+                 If <parent-hash> and <tree-init-hash> are not provided, we
+                 create a new issue.
+                 If <parent-hash> is provided and <tree-init-hash> is not, we
+                 create a new issue which is attached to an existing commit.
+                 That can be used (for example) to attach a bug report to a
+                 commit.
+                 If both are provided, we create a reply to an existing issue
+                 discussion.
```

Comment:
I presume you copied this from the comment at the top of the script. Please use the help text instead. The functionality has changes somewhat. (I apparently forgot to update the comment)